src/style.css

```diff
@@ -0,0 +1,126 @@
+@import
```

Comment:
You duplicate styles in this file. Why? You've already moved styles to separate files.